### PR TITLE
Refactor docs struct and content.

### PR DESCRIPTION
**Pull request by shibd**

### CODE REVIEW

File: docs/aws-eventbridge-sink-new.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+# Limitations
+- The Amazon AWS EventBridge and Pulsar cluster should be in the same cloud and region. (If the regions are not together, it will affect the synchronization efficiency)
+- The connector just support `AVRO`, `JSON`, `PROTOBUF_NATIVE` schema for Pulsar. Please refer [Data convert](## Data convert).
+
+
+# Quick start
+
+> Before start, Make sure you have completed **Prerequisites**.
+ 
+## 1. Create Connector
```
Comment (shibd) [inline docs/aws-eventbridge-sink-new.md:69]: When the user comes to this step,Needs to be carried out in conjunction with the reading `sn cloud`, `function mesh`, `pulsar function worker` docs.

**Comment by momo-jun** [comment]
@shibd, thanks for the initiative to restructure the docs.

I have the following suggestions for the proposed structure in general:
1. Make "Prerequisites" relevant to an action (user task) to provide more context, for example, the Quick Start for setting up the connector. Does it make sense to move "Prerequisites" under "Quick Start"?
2. No need to amplify "limitations". The two items you listed here are actually the deployment requirement and supportability, IMO. We can eliminate the heading "limitations" and attach the bullets to where it applies.

**Review by poorbarcode — APPROVED** [review]

File: docs/aws-eventbridge-sink-new.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+
+To create AWS EventBridge sink connector, depending on the environment, there are several ways to create it:
+
+- [Create Connector with StreamNative Cloud](https://main.sn-docs.pages.dev/docs/connector-pulsarctl).
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink-new.md:73]:
```suggestion
- [Create Connector on StreamNative Cloud](https://docs.streamnative.io/docs/connector-create).
```

You can use the link for the new doc.

File: docs/aws-eventbridge-sink-new.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+   }
+}
+```
+> * The above different creation methods are different for the structure of the configuration, 
+>  some are **json**, some are **yaml**, some are **k8s yaml**, you only need to replace with the corresponding format.
+>  
+> * If you want to configure more, please refer to **Configuration Properties**
+
+## 2. Send message to topic
+> If your connector is created on StreamNative Cloud, please refer to this [document](TODO) on how to authenticate clients.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink-new.md:97]: Does this imply that users can skip this step if the connector is created with Function worker/mesh?

Comment (shibd) [reply docs/aws-eventbridge-sink-new.md:97]: No, Using the client on SN cloud requires some additional authentication configuration, and this is just a reminder to the user.

**Comment by shibd** [comment]
> Make "Prerequisites" relevant to an action (user task) to provide more context, for example, the Quick Start for setting up the connector. Does it make sense to move "Prerequisites" under "Quick Start"?

No, If the user does not do `Prerequisites`, they cannot "Quick Start".


> No need to amplify "limitations". The two items you listed here are actually the deployment requirement and supportability, IMO. We can eliminate the heading "limitations" and attach the bullets to where it applies as notes.

Make sense, I removed it.

**Comment by shibd** [comment]
> @shibd By nesting "Prerequisites" under "Quick Start"`, we can make it part of quickstart in a more cohesive way. They are the prerequisites of creating a connector, like any other sections in parallel.

I see. Make sense to me. Thanks explain.

File: docs/aws-eventbridge-sink.md [hunk 4]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+| `batchMaxTimeMs`        | long   | no       | 5000              | Batch max wait time: milliseconds.                                                                                                                                                                                                       |
+| `maxRetryCount`         | long   | no       | 100               | Maximum number of retries to send events, when put events failed.                                                                                                                                                                        |
+| `intervalRetryTimeMs`   | long   | no       | 1000              | The interval time(milliseconds) for each retry, when the put events failed.                                                                                                                                                              |
+
+> For advanced features and configuration of this connector, please refer to the Advanced feature for more information.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:172]:
```suggestion
> For details about this connector's advanced features and configurations, see [Advanced features](#advanced-features).
```

File: docs/aws-eventbridge-sink.md [hunk 5]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+
+## Advanced feature
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:174]:
```suggestion
## Advanced features
```

File: docs/aws-eventbridge-sink.md [hunk 6]
```diff
@@ -170,351 +329,7 @@ maxRetryCount: 100 // Maximum retry send event count, when event send failed.
...
-
-    ```
-    NAME                                         READY   STATUS      RESTARTS   AGE
-    pod/aws-eventbridge-sink-sample-0               1/1    Running     0          77s
-    ```
-
-   After that, you can produce and consume messages using the AWS EventBridge sink connector between Pulsar and
-   EventBridge.
-
+## More Link
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:332]:
```suggestion
## More links
```

File: docs/aws-eventbridge-sink.md [hunk 7]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+| `stsEndpoint`     | String | yes      | "" (empty string) | The AWS STS endpoint. By default, the default STS endpoint: https://sts.amazonaws.com is used. See [Amazon documentation](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html) for more details. |
+| `stsRegion`       | String | yes      | "" (empty string) | The AWS STS region, By default, the 'region' config or env region is used.                                                                                                                                    |
+| `role`            | String | yes      | "" (empty string) | The AWS STS [role.](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)                                                                                                                  |
+| `roleSessionName` | String | yes      | "" (empty string) | The AWS STS [role session name.](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)                                                                                                     |
+
+- Other Configuration
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:157]:
```suggestion
- Other configurations
```

File: docs/aws-eventbridge-sink.md [hunk 8]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+  }
+}
+```
+
+## Configuration Properties
+
+Before using the AWS EventBridge sink connector, you need to configure it. This table outlines the properties and the
+descriptions.
+
+This connector provider two authentication methods for AWS EventBridge, please select either authentication method from `Access keys` and `Security Token Service`.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:139]:
```suggestion
This connector provides two authentication methods for AWS EventBridge. You can select either one between `Access keys` and `Security Token Service`.
```

File: docs/aws-eventbridge-sink.md [hunk 9]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+   }
+}
+```
+> * The above different creation methods are different for the structure of the configuration, 
+>  some are **json**, some are **yaml**, some are **k8s yaml**, you only need to replace with the corresponding format.
+>  
+> * If you want to configure more, please refer to **Configuration Properties**
+
+### 2. Send message to topic
+> If your connector is created on StreamNative Cloud, please refer to this [document](TODO) on how to authenticate clients.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:91]:
```suggestion
> **Note:** If your connector is created on StreamNative Cloud, you need to authenticate your clients. See [Build applications using Pulsar clients](https://docs.streamnative.io/docs/qs-connect#jumpstart-for-beginners) for more information.
```

File: docs/aws-eventbridge-sink.md [hunk 10]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+      "eventBusName": "test-event-bus-name"
+   }
+}
+```
+> * The above different creation methods are different for the structure of the configuration, 
+>  some are **json**, some are **yaml**, some are **k8s yaml**, you only need to replace with the corresponding format.
+>  
+> * If you want to configure more, please refer to **Configuration Properties**
+
+### 2. Send message to topic
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:90]:
```suggestion
### 2. Send messages to the topic
```

File: docs/aws-eventbridge-sink.md [hunk 11]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+```json
+   "configs": {
+      "accessKeyId": "{{Your access access key}}",
+      "secretAccessKey": "{{Your secret access key}}",
+      "region": "test-region",
+      "eventBusName": "test-event-bus-name"
+   }
+}
+```
+> * The above different creation methods are different for the structure of the configuration,
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:85]:
```suggestion
> * The configuration structure varies depending on how you create the AWS EventBridge sink connector.
```

File: docs/aws-eventbridge-sink.md [hunk 12]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+   "configs": {
+      "accessKeyId": "{{Your access access key}}",
+      "secretAccessKey": "{{Your secret access key}}",
+      "region": "test-region",
+      "eventBusName": "test-event-bus-name"
+   }
+}
+```
+> * The above different creation methods are different for the structure of the configuration, 
+>  some are **json**, some are **yaml**, some are **k8s yaml**, you only need to replace with the corresponding format.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:86]:
```suggestion
>  For example, some are **JSON**, some are **YAML**, and some are **Kubernetes YAML**. You need to adapt the configs to the corresponding format.
```

File: docs/aws-eventbridge-sink.md [hunk 13]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+To create AWS EventBridge sink connector, depending on the environment, there are several ways to create it:
+
+- [Create Connector on StreamNative Cloud](https://docs.streamnative.io/docs/connector-create).
+- [Create Connector with Function worker](https://pulsar.apache.org/docs/3.0.x/io-quickstart/). 
+  This way need download **nar** package to create build-in or no-build-in connector. You can download the version you need from [here](https://github.com/streamnative/pulsar-io-aws-eventbridge/releases)
+- [Create Connector with Function mesh](https://functionmesh.io/docs/connectors/run-connector).
+  This way need set docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-aws-eventbridge/tags)
+
+Please go to the corresponding document for details. 
+Each creation way is similar, you just need to set the following parameters to the connector configuration.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:74]:
```suggestion
No matter how you create an AWS EventBridge sink connector, the minimum configuration contains the following parameters.
```

File: docs/aws-eventbridge-sink.md [hunk 14]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+
+To create AWS EventBridge sink connector, depending on the environment, there are several ways to create it:
+
+- [Create Connector on StreamNative Cloud](https://docs.streamnative.io/docs/connector-create).
+- [Create Connector with Function worker](https://pulsar.apache.org/docs/3.0.x/io-quickstart/). 
+  This way need download **nar** package to create build-in or no-build-in connector. You can download the version you need from [here](https://github.com/streamnative/pulsar-io-aws-eventbridge/releases)
+- [Create Connector with Function mesh](https://functionmesh.io/docs/connectors/run-connector).
+  This way need set docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-aws-eventbridge/tags)
+
+Please go to the corresponding document for details.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:73]: We can remove this sentence. It's explicit.

File: docs/aws-eventbridge-sink.md [hunk 15]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+ 
+### 1. Create Connector
+
+To create AWS EventBridge sink connector, depending on the environment, there are several ways to create it:
+
+- [Create Connector on StreamNative Cloud](https://docs.streamnative.io/docs/connector-create).
+- [Create Connector with Function worker](https://pulsar.apache.org/docs/3.0.x/io-quickstart/). 
+  This way need download **nar** package to create build-in or no-build-in connector. You can download the version you need from [here](https://github.com/streamnative/pulsar-io-aws-eventbridge/releases)
+- [Create Connector with Function mesh](https://functionmesh.io/docs/connectors/run-connector).
+  This way need set docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-aws-eventbridge/tags)
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:71]:
```suggestion
  Using this way requires you to set the docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-aws-eventbridge/tags)
```

File: docs/aws-eventbridge-sink.md [hunk 16]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+- Set the target according to your needs. If you're testing this connector, you can set target to [Cloud Watch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html)
+
+ 
+### 1. Create Connector
+
+To create AWS EventBridge sink connector, depending on the environment, there are several ways to create it:
+
+- [Create Connector on StreamNative Cloud](https://docs.streamnative.io/docs/connector-create).
+- [Create Connector with Function worker](https://pulsar.apache.org/docs/3.0.x/io-quickstart/). 
+  This way need download **nar** package to create build-in or no-build-in connector. You can download the version you need from [here](https://github.com/streamnative/pulsar-io-aws-eventbridge/releases)
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:69]:
```suggestion
  Using this way requires you to download a **NAR** package to create a built-in or non-built-in connector. You can download the version you need from [here](https://github.com/streamnative/pulsar-io-aws-eventbridge/releases).
```

File: docs/aws-eventbridge-sink.md [hunk 17]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+}
+```
+4. Create a [Rule](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-create-rule.html) in EventBridge.
+- The data structure sent to Event Bridge is described [here](## Metadata mapping), and you can create **event pattern** based on this structure.
+- Set the target according to your needs. If you're testing this connector, you can set target to [Cloud Watch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html)
+
+ 
+### 1. Create Connector
+
+To create AWS EventBridge sink connector, depending on the environment, there are several ways to create it:
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:65]:
```suggestion
Depending on the environment, there are several ways to create an AWS EventBridge sink connector:
```

File: docs/aws-eventbridge-sink.md [hunk 18]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+      }
+    ]
+}
+```
+4. Create a [Rule](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-create-rule.html) in EventBridge.
+- The data structure sent to Event Bridge is described [here](## Metadata mapping), and you can create **event pattern** based on this structure.
+- Set the target according to your needs. If you're testing this connector, you can set target to [Cloud Watch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html)
+
+ 
+### 1. Create Connector
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:63]:
```suggestion
### 1. Create a connector
```

File: docs/aws-eventbridge-sink.md [hunk 19]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
+            "aws:PrincipalOrgID": "o-yourOrgID"
+          }
+        }
+      }
+    ]
+}
+```
+4. Create a [Rule](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-create-rule.html) in EventBridge.
+- The data structure sent to Event Bridge is described [here](## Metadata mapping), and you can create **event pattern** based on this structure.
+- Set the target according to your needs. If you're testing this connector, you can set target to [Cloud Watch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html)
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:60]:
```suggestion
- Set the target according to your needs. If you're testing this connector, you can set the target to [Cloud Watch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html).
```

File: docs/aws-eventbridge-sink.md [hunk 20]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
-This section describes the features of the AWS EventBridge sink connector. For details about how to configure these
-features, see [how to configure](#how-to-configure).
+### Prerequisites
+
+Prerequisites for an AWS EventBridge sink connector connecting to external systems:
+
+1. Create EventBridge and EventBus in AWS.
+2. Create the AWS authentication key, which supports two authentication methods:
+- [Access Keys](https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html): If you use this method, Just need set `accessKeyId` and `secretAccessKey` on this connector config.
+- [Security Token Service](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html): If you use this method, Just need set `stsEndpoint`, `stsRegion`, `role`, and `roleSessionName` on this connector config.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:20]:
```suggestion
- [Security Token Service](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html): To use this method, you need to set `stsEndpoint`, `stsRegion`, `role`, and `roleSessionName` on this connector config.
```

File: docs/aws-eventbridge-sink.md [hunk 21]
```diff
@@ -8,20 +8,180 @@ data to Amazon EventBridge.
...
 
-This section describes the features of the AWS EventBridge sink connector. For details about how to configure these
-features, see [how to configure](#how-to-configure).
+### Prerequisites
+
+Prerequisites for an AWS EventBridge sink connector connecting to external systems:
+
+1. Create EventBridge and EventBus in AWS.
+2. Create the AWS authentication key, which supports two authentication methods:
+- [Access Keys](https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html): If you use this method, Just need set `accessKeyId` and `secretAccessKey` on this connector config.
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:19]:
```suggestion
- [Access Keys](https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html): To use this method, you need to set `accessKeyId` and `secretAccessKey` on this connector config.
```

**Review by momo-jun — CHANGES_REQUESTED** [review]
Added a few suggestions to fix obvious typos and align the common writing style.

File: docs/aws-eventbridge-sink.md [hunk 22]
```diff
@@ -8,20 +8,166 @@ data to Amazon EventBridge.
...
 
-This section describes the features of the AWS EventBridge sink connector. For details about how to configure these
-features, see [how to configure](#how-to-configure).
+### Prerequisites
+
+Prerequisites for an AWS EventBridge sink connector connecting to external systems:
 
-## Delivery guarantees
+1. Create EventBridge and EventBus in AWS.
+2. Create the [AWS User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) and create `Access Key`(Please record `accessKey` and `secretAccessKey`).
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:18]: Any difference between `Access Key` and `accessKey`? Does it make sense to get them aligned?

File: docs/aws-eventbridge-sink.md [hunk 23]
```diff
@@ -8,20 +8,166 @@ data to Amazon EventBridge.
...
+3. Set permissions to users, ensure that the user has the following permissions to the AWS EventBus. For details, see [permissions for event buses](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-bus-perms.html)
```
Comment (momo-jun) [inline docs/aws-eventbridge-sink.md:19]:
```suggestion
3. Assign permissions to Pulsar users, and ensure they have the following permissions to the AWS EventBus. For details, see [permissions for event buses](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-bus-perms.html)
```

**Review by RobertIndie — APPROVED** [review]

**Pull request merged**